packages/block-editor/src/components/inserter/menu.js

```diff
@@ -92,6 +92,11 @@ function InserterMenu( {
 			</div>
 			{ showInserterHelpPanel && (
 				<div className="block-editor-inserter__tips">
+					<VisuallyHidden>
+						<h2 className="block-editor-inserter__tips-title">
+							{ __( 'Tips for using the block editor:' ) }
```

Comment:
Thanks for the PR 👍 I think we should probably remove the ":" and we may want to reword that a little bit. I'll add a "copy review" label.

Reply:
Thanks - should I wait for the copy review before updating the PR without the `:` or should I change it right now?

Reply:
Yes, let's wait a bit and do a single update.

Reply:
Since only one tip is shown at a time, I think it should say "Tip for using the block editor:".

Reply:
For the copy, can we just say "Block editor tip!"

Reply:
@michelleweber Remember, this is screen-reader-only text. I think adding an exclamation point might make the screen reader say it with a different inflection, which may not be desirable. (Then again, I've never used a screen reader, so I have no idea.)

Reply:
Ah, gotcha. In that case, I say stick with "A tip for using the block editor:"

(With whatever final punctuation makes the most sense for a screen reader.)